app/models/shipping_co.rb

```diff
@@ -1,10 +1,18 @@
 class ShippingCo < ApplicationRecord
   has_many :price_settings, dependent: :destroy
   has_many :employees, dependent: :restrict_with_error
+  has_many :reach_areas, dependent: :destroy
 
   validates :name, :corp_name, :cnpj, :address, :status, presence: true
   validates :name, :corp_name, :cnpj, uniqueness: true
   validates :cnpj, length: { is: 14 }
 
   enum status: { inactive: 0, active: 1 }
+
+  def list_reach_areas
+    return nil if reach_areas.empty?
```

Comment:
```suggestion
    return if reach_areas.empty?
```

Aqui poderia usar só um guardclause normal com retorno de nil implícito 😄

Reply:
hehe oops... Obrigada!